src/compositemark/boxplot.ts

```diff
@@ -47,6 +47,8 @@ export const BOXPLOT_STYLES: BoxPlotStyle[] = ['boxWhisker', 'box', 'boxMid'];
 export interface BoxPlotConfig extends MarkConfig {
   /** Size of the box and mid tick of a box plot */
   size?: number;
+  /** A scalar that will be multiplied by the IQR */
```

Comment:
Shouldn't this have the same description as `extent` in `BoxPlotDef`?

Reply:
No, I use this config extent property just when extent isn't specified in the mark def

Reply:
Yes, but say if you put "min-max" here it should work too.   So at least it should say about min-max here.